controllers/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -775,11 +756,18 @@ func (r *ReconcilePublishingStrategy) ensureIngressController(reqLogger logr.Log
 	// At this point, the IngressController doesn't exist anymore
 	// Create the desiredIngressController (hopefully before cluster-ingress-operator did)
 	reqLogger.Info(fmt.Sprintf("Create IngressController %s", ingressController.Name))
-	if err := r.client.Create(context.TODO(), desiredIngressController); err != nil {
+	if err := r.Client.Create(context.TODO(), desiredIngressController); err != nil {
 		reqLogger.Error(err, "Error creating the IngressController")
 		return reconcile.Result{Requeue: true}, err
 	}
 	// If the CR was created, requeue PublishingStrategy
 	return reconcile.Result{Requeue: true}, nil
 
 }
+
+// SetupWithManager sets up the controller with the Manager.
+func (r *PublishingStrategyReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewControllerManagedBy(mgr).
+		Owns(&v1alpha1.PublishingStrategy{}).
```

Comment:
Similarly to my earlier comment, we should check that this should not be `For` instead.

Reply:
Yeah, I think it should be `For` instead of `Owns`. 

Fixed, thanks!